client/app/src/accounts/AccountController.js

```diff
@@ -151,19 +150,13 @@
     self.connection = networkService.getConnection();
 
     self.connection.then(
-      function(){
-
-      },
-      function(){
-
-      },
```

Comment:
should be kept:
- first function: when promise is resolved
- second function: when promise is failed
- third one: when promise is changed